packages/cli/src/lib/decorators/preconditions/trackable.ts

```diff
@@ -1,25 +1,46 @@
 import type Command from '@oclif/command';
 import {buildError, buildEvent} from '../../../hooks/analytics/eventUtils';
 
-export function Trackable(overrideEventProperties?: Record<string, unknown>) {
+export interface TrackableOptions {
+  /**
+   * Event name used to identify the command.
+   * The expression should go as follows: **(Subject) (Process)?**
```

Comment:
Why the question mark at (Process)?

Reply:
Because it is optional

Reply:
👍 use the JSDOC standard for 'optionality' then: https://jsdoc.app/tags-param.html#optional-parameters-and-default-values
```suggestion
   * The expression should go as follows: **(Subject) [Process]?**
```